docs/contributing/feature-stages.md

```diff
@@ -53,11 +53,31 @@ coder server --experiments=feature1,feature2
 
 ## Beta
 
-Beta features are open to the public, but are tagged with a `Beta` label.
+- **Stable**: No
+- **Production-ready**: Not fully
```

Comment:
```suggestion
- **Production-ready**: Yes. Might not be feature-complete.
```

Reply:
Nice. I agree. We can also mention minor bugs may be present